app/controllers/review_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ReviewController < ApplicationController
+  def index
+    @reviews = Review.all
+  end
+
+  def show
+    @review = Review.find(params[:id])
+  end
+
+  def edit
+    @review = Review.find(params[:id])
+  end
+
+  def update
+    @review = Review.find(params[:id])
+    if @review.update_attributes(review_params)
+      # redirect and flash message
```

Comment:
Ditch unused commented code - or finish writing the `redirect` and `flash` message